todo_app/test_app.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from todo_app import app
+from dotenv import find_dotenv, load_dotenv
+from unittest.mock import patch, Mock
+
+TEST_BOARD_ID = '6006195ae74f187322fda449'
```

Comment:
Notice that your test is incorrectly dependent on this matching .env rather than .env.test

This is because flask_config.py is loading environment variables when your code is first imported, i.e. before your test has a chance to load .env.test.

You should update it to look more like this:
```
class Config:
    def __init__(self):
        self.SECRET_KEY = os.environ.get('SECRET_KEY')
        self.TRELLO_KEY= os.environ.get('TRELLO_KEY')
        self.TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
        self.TRELLO_BOARD_ID = os.environ.get('TRELLO_BOARD_ID')
```
And update the usage in app.py to `app.config.from_object(Config())`